SwiftReflector/CustomSwiftCompiler.cs

```diff
@@ -188,7 +188,7 @@ public Version GetCompilerVersion ()
 				locations.DisposeAll ();
 			}
 			ThrowOnCompilerVersionMismatch (output, moduleNames);
-			using (var stm = new FileStream (output, FileMode.Open, FileAccess.Read, FileShare.ReadWrite)) {
+			using (var stm = new FileStream (pathName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite)) {
```

Comment:
how is this related to the other changes ?

Reply:
Because I used this code to debug it and it's using the the error string as a path name. It didn't work, so I fixed it so I could debug.